webtau-http-groovy/src/test/groovy/com/twosigma/webtau/http/HttpGroovyTest.groovy

```diff
@@ -921,15 +868,15 @@ class HttpTest implements HttpConfiguration {
         return given
     }
 
-    private static TestServerJsonResponse jsonResponse(String resourceName, int statusCode = 200, Map headerResponse = [:]) {
-        return new TestServerJsonResponse(ResourceUtils.textContent(resourceName), statusCode, headerResponse)
-    }
-
     private static void assertStatusCodeMismatchRegistered() {
         http.lastValidationResult.mismatches.should contain(~/statusCode/)
     }
 
     private static byte[] binaryFile(String path) {
         return [1, 2, 3] as byte[]
     }
+
+    private static Path testResourcePath(String relativePath) {
+        return Paths.get("../webtau-http/src/test/resources/image.png")
```

Comment:
How is the build succeeding with this?  You're using `testResourcePath` in all the tests but ignoring its parameter so it's always doing a get for this png.

Reply:
Oh, I see, it's always passing in the same png param which happens to match what you actually do in this method.  Should fix this though, either use the param or don't ask for param and call the method `testImagePath()`.